### DISCUSSION TITLE
Great table lib!

### DISCUSSION THREAD

**Discussion by wuarmin** [table-library/react-table-library#5] [General]
Hey @rwieruch,
I'm having the same issue as you had: Looking for the right react-table component, which includes all features I need ;). I'm coming from ember.js and I have invested a lot of time in the search for the right table component. 
Now I i stumbled upon this library by accident (through https://www.robinwieruch.de/styled-components). I also have your book "Road to React" at home and read it a few years ago. 

react-table-library looks promising and because of this I have some questions:
Whats your goal with the library? At the moment, I think, just a few people know about it. 
Do you use the lib in production? If yes, do you recommend using it in prod to other people at this early stage? Are you the only maintainer? Do you have contributors?

One feature I would need is a fixed header.  In the ember.js ecosystem there is one great table-lib (https://github.com/Addepar/ember-table). It is very very complete and stable and I like it a lot. Do you think react-table-library could evolve similar.

Thank you and best regards!

**Comment by rwieruch** [comment]
Hey @wuarmin Thanks for your questions and interest.

There is no specific goal for this library. For the application where we use it in production, it serves its purpose without any bugs. If this application needs more features, we will most likely implement it in this library. Anyway, if more people would use it, I would see myself investing more time in it for feature requests etc. 

Yes, we are using it in production. At the moment, I am the only maintainer for it. A bit of background on why I built it: https://www.robinwieruch.de/react-tree-list/

A fixed header feature is included now: https://react-table-library.com

Let me know if you start using it :) I will definitely advertise the library more and more over the next weeks.
